widget/embedded/src/utils/wallets.ts

```diff
@@ -358,20 +358,6 @@ function numberWithThousandSeparator(number: string | number): string {
   return parts.join('.');
 }
 
-export const getUsdPrice = (
-  blockchain: string,
-  symbol: string,
-  address: string | null,
-  allTokens: Token[]
-): number | null => {
-  const token = allTokens?.find(
-    (t) =>
-      t.blockchain === blockchain &&
-      t.symbol?.toUpperCase() === symbol?.toUpperCase() &&
-      t.address === address
-  );
-  return token?.usdPrice || null;
-};
```

Comment:
This function iterates over all tokens to get their prices. I think you can modify it to use the tokenMap instead.

Reply:
fixed